donkeycar/templates/cfg_complete.py

```diff
@@ -221,6 +221,16 @@
 STEERING_LEFT_PWM = 460    # pwm value for full left steering (use `donkey calibrate` to measure value for your car)
 STEERING_RIGHT_PWM = 290   # pwm value for full right steering (use `donkey calibrate` to measure value for your car)
 
+#VESC controller, primarily need to change VESC_SERIAL_PORT  and VESC_MAX_SPEED_PERCENT
+VESC_MAX_SPEED_PERCENT =.2  # Max speed as a percent of the actual speed
+VESC_SERIAL_PORT= "/dev/ttyACM0" # Serial device to use for communication. Can check with ls /dev/tty*
+VESC_HAS_SENSOR= True # Whether or not the bldc motor is using a hall effect sensor
+VESC_START_HEARTBEAT= True # Whether or not to automatically start the heartbeat thread that will keep commands alive.
+VESC_BAUDRATE= 115200 # baudrate for the serial communication. Shouldn't need to change this.
+VESC_TIMEOUT= 0.05 # timeout for the serial communication
+VESC_STEERING_SCALE= 0.5 # VESC accepts steering inputs from 0 to 1. Joystick is usually -1 to 1. This changes it to -0.5 to 0.5
```

Comment:
I understand from the comments that VESC_STEERING_SCALE and VESC_STEERING_OFFSET work to get to scale and translate the joystick input from -1..1 to 0..1 so it is compatible with VESC.  My question is, should this even be configurable?  Can we just hard code this (or at assign default to the related arguments and don't expose any configuration).

Reply:
Mentioned in the other PR but this should be configurable since the web UI has a default mapping of 0 to 1 which is what the VESC expects (which means the scale and offset will both be 0)

Reply:
What web UI are you talking about?  If the http://<yourcar>.local::8887/drive is producing only 0..1 values for throttle then that is a bug.  It should act like a game controller and produce -1 to 1.